ENIGMAsystem/SHELL/Universal_System/Extensions/Asynchronous/ASYNCdialog.cpp

```diff
@@ -124,27 +124,27 @@ namespace enigma_user {
     return queue_async_job(fnc);
   }
 
-  int get_string_async(string message, string def, string cap) {
+  int get_string_async(string str, string def) {
```

Comment:
Why did you rename `message` to `str`? It's less descriptive.

Reply:
That's what GM uses, and that is what we are using in the Mac and Linux Systems.

Reply:
I'd sooner change it in those systems. It's very obvious what `message` means in this context, but naming a parameter to `get_string` just `str` is incredibly helpful. It would as well not have a name.

Reply:
I'll change it, but first I want to see if there are any other critiques you might have which I'll probably address all them sometime tmr.

Reply:
I'll make that change in a different pr directly after this one is merged, to keep the changes more organized.